app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    @staticmethod
+    def print_name(name: str) -> None:
+        print(f"Hello, I'm {name}")
+
+    @staticmethod
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = False
+        return self.appetite
```

Comment:
```suggestion
    def feed(self) -> int:
        if not self.is_hungry:
            return 0
        print(f"Eating {self.appetite} food points...")
        self.is_hungry = False
        return self.appetite
```
please fix flake8 error